howso.amlg

```diff
@@ -708,6 +708,62 @@
 			;return that this has completed
 			(call !return)
 		)
+	
+	;gets the model's auto-ablate parameters set by #set_auto_ablate_params
+	#get_auto_ablate_params
+		(call !return (assoc
+			payload
+				(call_entity trainee "GetAutoAblateParams")
+		))
+
+	;sets the model to auto-ablate by tracking its size and training certain cases as weights
+	;parameters
+	; auto_ablate_enabled: flag, default is false. when true, any enabled ablation techniques will be run at during training.
+	; auto_ablate_weight_feature: string, default is ".case_weight". the weight feature that should be used when ablating.
+	; minimum_model_size: optional, stores the threshold for the minimum number of cases at which the model should auto-ablate.
+	;                     default of 1000.
+	; influence_weight_entropy_threshold: optional, the influence weight entropy quantile that a case must be beneath in order to be trained.
+	;                                     default of 0.6.
+	; exact_prediction_features: optional list of features. for each of the features specified, will ablate a case if the prediction
+	;                            matches exactly.
+	; tolerance_prediction_threshold_map: optional assoc of feature -> [MIN, MAX]. for each of the features specified, will
+	;                                     ablate a case if the prediction >= (case value - MIN) and the prediction <= (case value + MAX)
+	; relative_prediction_threshold_map: optional assoc of feature -> PERCENT. for each of the features specified, will
+	;                                    ablate a case if abs(prediction - case value) / prediction <= PERCENT
+	; residual_conviction_features: optional list of features. for each of the features specified, will ablate a case if
+	;                               abs(prediction - case value) <= feature residual
+	; conviction_upper_threshold: optional float. the conviction value below which cases will be ablated
+	; conviction_lower_threshold: optional float. the conviction value above which cases will be ablated
+	#set_auto_ablate_params
```

Comment:
please add get and set_auto_ablate params to the list of api calls at the top of this file